qa/workunits/rbd/rbd_mirror.sh

```diff
@@ -219,94 +219,16 @@ admin_daemon()
     ceph --admin-daemon ${asok_file} $@
 }
 
-status()
```

Comment:
I can put it back -- I saw that it wasn't in-use within the script so I yanked it.